library/src/main/res/values/ids.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <item name="colorPrimary" type="color" />
+    <item name="colorPrimaryDark" type="color" />
+    <item name="colorAccent" type="color" />
```

Comment:
These colors are now required. We could add defaults if you'd like.

Reply:
I think defaults are not needed if we make sure to note that in README.

Reply:
Sweet!